README.md

```diff
@@ -43,6 +44,10 @@ and then run:
 
     jupyter nbconvert --config config.py demo.ipynb
 
+### 3rd party email distributor Example
+Insread of using SMTP to send emails, one can use 3rd party provider.  
```

Comment:
Instead.

Reply:
Typo is fixed with a new commit.
Thanks!